web/src/main/webapp/xslt/skin/default/skin.xsl

```diff
@@ -16,7 +16,6 @@
 
   <xsl:param name="output" as="xs:string" select="'not-pdf'"/>
```

Comment:
I know this is not part of your PR but I don't believe this is correct.

<xsl:param name="output" as="xs:string" select="'not-pdf'"/>

I think your original code of setting the value to "" was better.

Reason:  PDF is too specific. What happens if we decide to export to some other format i.e. exporting to Microsoft Word DOC/DOCX. In that case would the value be "not-pdf" && "not-doc"

Reply:
That is not part of this PR, but from this other PR: https://github.com/geonetwork/core-geonetwork/pull/7481/files#diff-0288f54a93a5cbb90245269c2610c967bea1bfedad167b40f57270c08f9e0cc1R17